README.md

```diff
@@ -40,13 +40,19 @@ private_movie_views = \
 
 # Calculate the private sum of ratings per movie
 dp_result = private_movie_views.sum(
-    SumParams(max_partitions_contributed=2,
-              max_contributions_per_partition=2,
+    SumParams(
+              # Limits to how much one user can contribute:
+              # .. at most two movies rated per user
+              max_partitions_contributed=2,
+              # .. at most one ratings for each movie
+              max_contributions_per_partition=1,
+              # .. with minimal rating of "1"
```

Comment:
Maybe to mention, that actual values will be clipped to min_value, max_value ?

Reply:
Done.